src/main/java/com/spikes2212/dashboard/Namespace.java

```diff
@@ -214,11 +215,11 @@ default PIDSettings addPIDNamespace(String name, PIDSettings initialPIDSettings)
      */
     default FeedForwardSettings addFeedForwardNamespace(String name, FeedForwardSettings initialFeedForwardSettings) {
```

Comment:
add an overloading method that receives a controlMode instead of ffSettings, and uses 0s as initial values for the gains